### ISSUE TITLE
The API ingestion menu fails for me and the chat opens in a window not the viewer pane

### ISSUE THREAD

**Issue by MichelNivard** [MichelNivard/gptstudio#50]
@JamesHWade 

1st bug I didnt notice, because I rarely use it, but the API ingestino menu fails for me, do we really need it? setting an environmental variable isn't such a major hassle right?

 Screenshot of it failing:

![bug](https://user-images.githubusercontent.com/11858442/222548502-7d38fe41-39e8-4813-b3c9-07ca9fc64efa.png)

2nd bug is it intentional that the chat screen opens in a popup and not the viewer pane? It default toa windowed view for me, perhaps thats a preference you had, maybe we could make it a variable (I could do that if you agree its valuable).

**Comment by JamesHWade** [comment]
API ingestion should be fixed with #51. I can't reproduce the opening in new window. I thought it was set to open in viewer pane. Here's the functions:

```{r}
chat_gpt_addin <- function() {
  check_api()
  withr::local_options(shiny.launch.browser = .rs.invokeShinyPaneViewer)
  run_chat_gpt()
}
```

**Comment by JamesHWade** [comment]
#51
